forgettable-frontend/src/components/IconButton/IconButton.js

```diff
@@ -3,17 +3,29 @@ import React from 'react';
 import {Button} from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
 
-function IconButton({btnText, onClick, includeIcon, height}) {
+const CUSTOM_ICON_SIZE = '20px';
+
+function IconButton({btnText, onClick, includeIcon, height, maxWidth, customIcon}) {
   const prmry = getComputedStyle(document.documentElement).getPropertyValue('--prmry');
   const txt3 = getComputedStyle(document.documentElement).getPropertyValue('--txt3');
   const fontMedium = getComputedStyle(document.documentElement).getPropertyValue('--font-medium');
   const textLarge = getComputedStyle(document.documentElement).getPropertyValue('--text-large');
   const radiusBtn = getComputedStyle(document.documentElement).getPropertyValue('--radius-btn');
 
+  let icon = <AddIcon />;
+  if (customIcon) {
+    icon = <img
+      src={customIcon}
+      alt="Button Icon"
+      height={CUSTOM_ICON_SIZE}
+      width={CUSTOM_ICON_SIZE}
```

Comment:
I would have thought it would be better to put this in the css files, but I guess you can't change them as easily? Is it somethin to do with the values getting overwritten if you put it in the css?